modules/lib/src/main/resources/i18n/phrases.properties

```diff
@@ -69,6 +69,7 @@ action.archiveMore=Archive...
 action.fold=Fold
 action.show.more=Show more
 action.show.less=Show less
+action.ok=Ok
```

Comment:
No need to add a new one. We already have it in lib-admin-ui's common.properties and they are concatenated with Content Studio's.